lib/rest_openshift.rb

```diff
@@ -31,75 +31,73 @@ def self.version(base_opts, opts)
         }
       end
 
-      def self.delete_oauthaccesstoken(base_opts, opts)
-        populate("/oauthaccesstokens/<token_to_delete>", base_opts, opts)
+      def self.delete_oauthaccesstoken(base_opts, opts, type="oauth")
+        populate("/oauthaccesstokens/<token_to_delete>", base_opts, opts, type)
         return perform(**base_opts, method: "DELETE")
       end
 
-      def self.list_projects(base_opts, opts)
-        populate("/projects", base_opts, opts)
+      def self.list_projects(base_opts, opts, type="project")
```

Comment:
I think that `type` here should not be a parameter but hardcoded when calling `populate` method. Why would we ever want to specify a different type when calling `list_projects`?

Reply:
yeah, we can put it here or at the `populate` method as you suggested

Reply:
Ok, please remove this parameter `type` from here as well other individual calls. `populate` looks good.

See my other comment. I would leave it to you about merging k8s and openshift calls in same place.

Reply:
@akostadinov ok, I fixed that now.